### ISSUE TITLE
languages.json documentation

### ISSUE THREAD

**Issue by gregpettit** [jquery-i18n-properties/jquery-i18n-properties#15]
I see that the plugin attempts to load {path}/languages.json ... yet I don't see any documentation detailing what this file should be or what it should look like. Have I missed part of the documentation?

**Comment by zmpandzmp** [comment]
I also wondered for a long time because of this problem,I think the version 1.0.9 is very good!

**Comment by albert-kam** [comment]
I checked within the source code, and here's my assumptions

Example of the format for languages.json would be 
{
  languages: ['en', 'pt_PT']
}

In this example, we only support these 2 languages

So if in the jQuery.i18n.properties we define the language as 'en', then Messages_en.properties will be loaded. If not defined in the languages.json, then it wont be loaded.

But if we dont need the extra languages support, and load only Messages.properties, the languages.json can be left empty, or even left uncreated.

Bottom line: If you're going with multiple languages, specify all the supported languages in languages.json

**Comment by bansalparijat** [comment]
Thanks a lot

**Comment by jrondorf** [comment]
I returned to the version **without** the languages.json implementation, as this was more fail proof in regards to missing files.

**Comment by agomezmoron** [comment]
I also returned to the 1.0.9 version. The languages.json option is powerful but should be optional, I mean: an user should be able to configure in the initialization if the system will use that option to reaed all the available languages or the other one. With the current version the backward compatibility is not working.

**Comment by dpocock** [comment]
can anybody propose a patch for this?

**Comment by dpocock** [comment]
@adrianfish - could you please comment on this issue?

**Comment by dpocock** [comment]
This is the pull request that introduced the change https://github.com/jquery-i18n-properties/jquery-i18n-properties/pull/13

**Comment by manueljordan** [comment]
Hello 

It has been fixed? About to get the `languages.json` file?

Currently the only way to avoid that error message is creating the `languages.json` file located together with all the .properties files with the following content in my case:

```
{
	"languages": ["en_US", "es_ES", "pt_PT"]
}
```

Thanks

**Comment by adrianfish** [comment]
I'll write some docs for it. Where should I write them? In the Wiki?

I'll also submit a patch to handle the lack of a languages file.

**Comment by manueljordan** [comment]
@adrianfish 

I think should be added on https://github.com/jquery-i18n-properties/jquery-i18n-properties in the  **Example** section, it because there appears the _.properties_ files. And the `languages.json` file must be located with them.

It should be added how a special note.

Thanks

**Comment by adrianfish** [comment]
@manueljordan Hi Manuel, could you have a quick look at the main README and see if the section I've added makes sense? Cheers.

**Comment by manueljordan** [comment]
Hi @adrianfish 

has sense and is OK….. Just one error 
Must be `pt_BR` no `pt_PT`. It was my mistake when I wrote my original post in this thread.
So in https://github.com/jquery-i18n-properties/jquery-i18n-properties replace/fix all the wrong apparitions of the `pt_PT` term/pattern. Sorry by my mistake.

Thanks

**Comment by manueljordan** [comment]
Furthermore on:
https://github.com/jquery-i18n-properties/jquery-i18n-properties/tree/master/bundle
Exists the `Messages_pt_PT.properties` file, it must be `Messages_pt_BR.properties`

**Comment by adrianfish** [comment]
So, you want the file Messages_pt_PT.properties renaming to Messages_pr_BR.properties?

**Comment by manueljordan** [comment]
Yes, That is the idea. The correct pattern is with `pr_BR`. It must be done for many apparitions on 

* https://github.com/jquery-i18n-properties/jquery-i18n-properties 
* https://github.com/jquery-i18n-properties/jquery-i18n-properties/tree/master/bundle

**Comment by adrianfish** [comment]
All done now, thanks for the heads up.

**Comment by manueljordan** [comment]
Something is wrong, on:

* https://github.com/jquery-i18n-properties/jquery-i18n-properties 

appears the wrong `pt_PT` pattern many times yet. Seems you must update the `README.md` file too. I've checked that file and there remains the problem yet.

**Comment by adrianfish** [comment]
Sorry, I was grepping on messages_pt. No instances of pt_PT now.

**Comment by manueljordan** [comment]
Again, on: 

* https://github.com/jquery-i18n-properties/jquery-i18n-properties

>>first and then locale specific files (Messages_pt.properties, then Messages_pt_BR.properties)

Appears `Messages_pt.properties` and is wrong

`Sequential loading of resource bundles from base language to user-specified/browser-specified so there is always a default value for an untranslated string (eg: msg.properties, msg_pt.properties, msg_pt_BR.properties)`

Appears `msg_pt.properties` and is wrong

furthermore:

```
Messages_pt.properties:      //<---- wrong

# We only provide a translation for the 'msg_hello' key
msg_hello = Bom dia

Messages_pt_PT.properties: //<---- wrong

# We only provide a translation for the 'msg_hello' key
msg_hello = Olá
```

**Comment by adrianfish** [comment]
I've just pushed another change to the README. Markdown syntax caught me out this time.
